docs/report/report.tex

```diff
@@ -385,14 +385,60 @@ \subsubsection{Retrospektive}
+Über die Entwicklungszeit wurden aber die neuen Technologien vertieft und verinnerlicht.
+Das Team lernte auch wieso Frameworks und Librarys wie Vue, Tailwind und Feathers sinvoll und hilfreich sein können.
+
+\subsubsection{Versionskontrolle und Deployment Cycle}
+Im Studium wurde den Studierenden der Vorteil von Versionskontrollsystemen wie Git vermittelt, doch fand sich im Studium selten ein praktisches Anwendungszenario, dass Git über die Hauptfunktion der Versionierung hinaus sinnvoll genutzt hat.
+Durch dieses Projekt fand sich das erste mal im Studium eine sinnvolle Anwendung, wie in größeren Projekten oder in der Industrie auch mit Pull Requests und Code Reviews zu arbeiten.
+Dadurch traten mehrere Effekte auf:
+Wenn mehrere Teammitglieder über den Code anderer schauten, fielen umso wahrscheinlicher Fehler auf und eine insgesamt bessere Codequalität wurde sichergestellt.
+Code Reviews hatten außerdem den Effekt, dass Teammitglieder neue Funktionen und Änderungen kennenlernen, verstehen und gegebenenfalls auch direkt Fragen dazu stellen konnten.
+Sich an diesen Deployment Cycle zu gewöhnen brauchte eine gewisse Zeit, aber fand von Sprint zu Sprint immer mehr Wertschätzung im Team.
+
+\subsubsection{Automatisierte Tests und Deployment}
+Jeder Pull Request unterlief automatisch auch mehrere automatisierte Tests, welche mit eigenen Testparametern über den neuen Quellcode lief.
+Durch diese Tests wurden sowohl dem Team viel manuelle Arbeit abgenommen, als auch wurde die Fehlerquelle Mensch minimiert.
+Neue Änderungen im Master wurden automatisch auf Github.io veröffentlicht, als auch wurden die \textit{LaTeX} Dateien als PDF im eigenen Dokumente Branch gebaut.
+Für Pull Requests wurde auch die jeweilige Version gebaut und auf einer \colorbox{yellow}{temporären} Webseite veröffentlicht.
+Dadurch konnte sich das Team mehr auf das Weiterentwickeln des Projektes konzentrieren.
+
+\subsubsection{Regelmäßige Standups und Retrospektiven}
+Durch die regelmäßigen Standups entstand ein Ort für das gegenseitige ansprechen von Problemen, updaten von Fortschritten und neuen Funktionen.
+So gingen seltener Probleme unter, neues Wissen wurde schneller verbreitet und die gesamte Teamproduktivität wurde verbessert.
+Das bei den Retrospektiven entstandene Feedback zwischen dem Teammitgliedern war sehr wichtig für das Teamklima.
+Auch die hierbei angesprochenen Probleme und Verbesserungsvorschläge wurden als sehr wertvoll vom Team für die nächsten Sprints erkannt.
+
+\subsubsection{Kommunikation und Eigeninitiative}
+Die Standups hatten auch den Effekt, dass das Team anfangs sich zu sehr auf diesen Kommunikationsweg verlassen hat, wodurch Probleme und wichtige Kommunikation bei Abhängigkeit zwischen Tickets zu selten statt fand.
+Hier fehlte anfangs die Eigeninitiative, bei Problemen aktiv sich Hilfe zu holen, auch weil manche Tickets zu schwierig wirkten, um diese alleine zu bewältigen.
```

Comment:
```suggestion
Hier fehlte anfangs die Eigeninitiative, sich bei Problemen aktiv Hilfe zu holen, auch weil manche Tickets zu komplex wirkten, um diese alleine zu bearbeiten.
```

Der Grund erschließt sich mir nicht ganz: Die Eigentinitiative fehlte, weil die Tickets zu schwierig waren?
Fragt man dann nicht eigentlich eher nach 😄 ?

Reply:
Anfangs wurden manche Tickets einfach nicht bearbeitet und es wurde etwas anderes oder gar nichts gemacht, oder so änhlich, you know?

Reply:
Ja, ich verstehe was du meinst. Das geht aber für mich nicht aus dem Text hervor. 

Dann wäre davor ein Satz noch gut der ausführt, dass die Tickets teilweise zu schwer wirkten und die Eigenititative ....blub blub blub..

Reply:
hab das mal umgesetzt in https://github.com/project-leek/project-leek/pull/419/commits/52e5df9d747132c37ebb8bb185740b747916d75b